Extensions/Physics2Behavior/physics2runtimebehavior.ts

```diff
@@ -166,17 +166,18 @@ namespace gdjs {
 
     step(deltaTime: float): void {
       this.frameTime += deltaTime;
-      if (this.frameTime >= this.timeStep) {
-        let numberOfSteps = Math.floor(this.frameTime / this.timeStep);
-        this.frameTime -= numberOfSteps * this.timeStep;
-        if (numberOfSteps > 5) {
-          numberOfSteps = 5;
-        }
-        for (let i = 0; i < numberOfSteps; i++) {
-          this.world.Step(this.timeStep * this.timeScale, 8, 10);
-        }
-        this.world.ClearForces();
-      }
+      // `frameTime` can take negative values.
+      // It's better to be a bit early rather than skipping a frame and being
+      // a lot more late.
+      let numberOfSteps = Math.round(this.frameTime / this.timeStep);
```

Comment:
Can this be negative? Should it be `Math.max(0, Math.round(`?

Reply:
I understand frameTime can be negative, but numberOfSteps should 0 or more, and I'm afraid that somehow we end up with frameTime being something like -2/60, so if timeStep is 1/60, we end up with `numberOfSteps == -2`

Reply:
Maybe it's not possible though as we always remove `numberOfSteps * timeStep`, and the round will round toward 0... but I want to be sure that it can't be negative. deltaTime could be really small and not sufficient to round toward 0.